dev/reference/tools/tidb-control.md

```diff
@@ -88,35 +94,37 @@ TiDB Controller 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，
 
 结果将以 json 形式展示，内容较长，这里做了截断。
 
-如希望指定表名，可以使用 `tidb-ctl schema in {数据库名} -n {表名}` 进行过滤。
+* 如希望指定表名，可以使用 `tidb-ctl schema in {数据库名} -n {表名}` 进行过滤。
```

Comment:
It seems that there is not another list item that is parallel with this one. Perhaps we should keep the original format.